modules/tinymce/src/themes/silver/main/ts/ui/core/Context.ts

```diff
@@ -20,6 +20,10 @@ const register = (editor: Editor): void => {
   editor.ui.registry.addContext('insert', (child: string) => {
     return editor.schema.isValidChild(editor.selection.getNode().tagName, child);
   });
+
+  editor.ui.registry.addContext('!readonly', () => {
+    return editor.mode.get() !== 'readonly';
+  });
```

Comment:
```suggestion
  editor.ui.registry.addContext('mode:!readonly', () => {
    return editor.mode.get() !== 'readonly';
  });
```
Would this fit into mode contexts?

Reply:
Pushed changes 👍  appreciate if this can be looked into!